routers/user/home.go

```diff
@@ -83,11 +83,11 @@ func retrieveFeeds(ctx *middleware.Context, ctxUserID, userID, offset int64, isP
 }
 
 func Dashboard(ctx *middleware.Context) {
-	ctx.Data["Title"] = ctx.Tr("dashboard")
+  ctxUser := getDashboardContextUser(ctx)
+	ctx.Data["Title"] = ctxUser.DisplayName() + " " + ctx.Tr("dashboard")
```

Comment:
Only you can see the dashboard for your account, and the page UI is almost 99% different, I think it's not a problem.